webapi/_test_data/ontologies/biblio-onto.ttl

```diff
@@ -92,12 +92,12 @@
 
                     rdfs:comment """Represents the location of a publication."""@en ;
 
+                    knora-base:subjectClassConstraint knora-base:Resource ;
```

Comment:
Why did you change this from `:Publication` to `:Resource`?

Reply:
hmm because I had used this property in Newton ontology for another resource. Then I have got rid of that resource, but this subjectClassConstraint has remained like this.  :/

Reply:
I think it's clearer to make it more specific if you can.

Reply:
done!